test/api/connectors/ConnectorSpec.scala

```diff
@@ -199,7 +198,7 @@ trait ConnectorSpec extends UnitSpec with Status with MimeTypes with HeaderNames
     MockedAppConfig.release6Environment returns "release6-environment"
     MockedAppConfig.release6EnvironmentHeaders returns Some(allowedIfsHeaders)
 
-    MockedAppConfig.featureSwitches returns Configuration("tys-api.enabled" -> false)
+    MockFeatureSwitches.isTaxYearSpecificApiEnabled.returns(true).anyNumberOfTimes()
```

Comment:
Was false before; should be false now:
```suggestion
    MockFeatureSwitches.isTaxYearSpecificApiEnabled.returns(false).anyNumberOfTimes()
```